ksp_plugin/plugin.cpp

```diff
@@ -445,10 +419,18 @@ void Plugin::SetPredictionLength(Time const& t) {
 
 void Plugin::SetPredictionLengthTolerance(Length const& l) {
   prediction_parameters_.set_length_integration_tolerance(l);
+  for (auto const& pair : vessels_) {
```

Comment:
This probably means that what we want is to have vessel-specific prediction parameters, and that eventually SetPredictionLengthTolerance should take a vessel. #957.